src/Network/Network/help/Remove-AzBastion.md

```diff
@@ -0,0 +1,218 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
```

Comment:
Please fill out `online version` in all new markdown files. Correct format is `https://docs.microsoft.com/en-us/powershell/module/az.network/` + cmdlet name in lowercase (e.g. `add-azdelegation`)

Reply:
added online version. Thanks